File: Android_Zone_Test/src/com/example/mylib_test/activity/file/Activity1.java
package com.example.mylib_test.activity.file;

import android.app.Activity;
import android.content.Intent;

public class Activity1 extends Activity{
	
	private boolean condition=true;
	@Override
	protected void onStart() {
		super.onStart();
		System.out.println("Activity1====onStart");
	}
	@Override
	protected void onResume() {
		super.onResume();
		System.out.println("Activity1====onResume");
		if (condition) {
			startActivity(new Intent(this, Activity2.class));
			condition=false;
		}
	}
	@Override
	protected void onStop() {
		super.onStop();
		System.out.println("Activity1====onStop");
	}
	@Override
	protected void onPause() {
		super.onPause();
		System.out.println("Activity1====onPause");
	}
	@Override
	protected void onRestart() {
		super.onRestart();
		System.out.println("Activity1====onRestart");
	}

}

File: Android_Zone_Test/src/com/example/mylib_test/activity/file/Activity1.kt
package com.example.mylib_test.activity.file

import android.content.Intent
import com.zone.lib.base.controller.activity.BaseFeatureActivity

class Activity1 : BaseFeatureActivity() {

    override fun setContentView() {
    }

    override fun initData() {
    }

    override fun setListener() {
    }

    private var condition = true
    override fun onStart() {
        super.onStart()
        println("Activity1====onStart")
    }

    override fun onResume() {
        super.onResume()
        println("Activity1====onResume")
        if (condition) {
            startActivity(Intent(this, Activity2::class.java))
            condition = false
        }
    }

    override fun onStop() {
        super.onStop()
        println("Activity1====onStop")
    }

    override fun onPause() {
        super.onPause()
        println("Activity1====onPause")
    }

    override fun onRestart() {
        super.onRestart()
        println("Activity1====onRestart")
    }

}
